src/test/java/com/conveyal/datatools/TestUtils.java

```diff
@@ -1,12 +1,100 @@
 package com.conveyal.datatools;
 
+import com.conveyal.datatools.manager.jobs.ProcessSingleFeedJob;
+import com.conveyal.datatools.manager.models.FeedSource;
+import com.conveyal.datatools.manager.models.FeedVersion;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static com.conveyal.datatools.manager.DataManager.GTFS_DATA_SOURCE;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
 public class TestUtils {
+
+    private static final Logger LOG = LoggerFactory.getLogger(TestUtils.class);
+
+    /**
+     * Returns true only if an environment variable exists and is set to "true".
+     */
     public static boolean getBooleanEnvVar (String var) {
-        String CI = System.getenv(var);
-        return CI != null && CI.equals("true");
+        String variable = System.getenv(var);
+        return variable != null && variable.equals("true");
     }
 
+    /**
+     * Checks whether the current environment appears to be a continuous integration environment.
+     */
     public static boolean isCi () {
         return getBooleanEnvVar("CI");
     }
+
+    /**
+     * Parse a json string into an unmapped JsonNode object
+     */
+    public static JsonNode parseJson(String jsonString) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
The Jackson docs recommend re-using heavyweight objects like ObjectMapper instances: https://github.com/FasterXML/jackson-docs/wiki/Presentation:-Jackson-Performance#basics-things-you-should-do-anyway